src/layout/SigneeList/SigneeListError.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { isAxiosError } from 'axios';
+import { z, ZodError } from 'zod';
+
+import { Lang } from 'src/features/language/Lang';
+import { useLanguage } from 'src/features/language/useLanguage';
+
+const problemDetailsSchema = z.object({
+  detail: z.string(),
+  status: z.number(),
+  title: z.string(),
+});
+
+export function SigneeListError({ error }: { error: Error }) {
+  const { langAsString } = useLanguage();
+
+  if (error instanceof ZodError) {
+    //   // TODO: alarm? telemetri?
+    window.logErrorOnce(
+      `Did not get the expected response from the server. The response didn't match the expected schema: \n${error}`,
+    );
+
+    return (
+      <div>
+        <Lang id='signee_list.parse_error' />
+        <br />
+        <Lang
+          id='general.customer_service_error_message'
+          params={[
+            'general.customer_service_phone_number',
+            'general.customer_service_email',
+            'general.customer_service_slack',
+          ].map((it, idx) => (
+            <Lang
+              key={idx}
+              id={it?.toString()}
+            />
+          ))}
```

Comment:
Why map a list of harcoded values instead of just harcoding the `<Lang />` components directly?

I think it should also be possible to add language references in params directly like:
```ts
params={[
  { key: 'general.customer_service_phone_number' },
  ...
]}
```